mc/Voyage-Mongo-Core.package/VOMongoConnectionError.class/properties.json

```diff
@@ -6,7 +6,7 @@
 		 ],
 	"commentStamp" : "<historical>",
 	"instvars" : [
-		 ],
+		"internalErrorsByTimestamp" ],
```

Comment:
not sure if collecting internal errors is useful; it was intended for debugging reasons. It does not represent overhead when no errors, since it's lazily initialized, but it's not really needed. I can remove this feature in a new commit.